src/components/SearchInput/SearchInput.stories.args.ts

```diff
@@ -26,4 +27,13 @@ export default {
       category: 'Props',
     },
   },
+  height: {
+    description: 'Changes the height of the component',
+    options: [...Object.values(HEIGHTS)],
```

Comment:
If `HEIGHT` is an array you can do this:
```suggestion
    options: HEIGHTS,
```